testsuite/tests/typing-misc/labels.ml

```diff
@@ -172,7 +172,8 @@ Line 1, characters 26-33:
                               ^^^^^^^
 Error: This expression has type "x:'a -> unit"
        but an expression was expected of type "unit -> unit"
-       The argument "x" is labeled, but an unlabeled argument was expected
+       The first argument is labeled "x",
+       but an unlabeled argument was expected
```

Comment:
Side remark: the absence of a final `.` at the end of these captialized sentences is odd, but this is orthogonal to the current change.

Reply:
I agree and generally struggle to remember to not add period in those explanations.